types/parse-multipart/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for parse-multipart 1.0
+// Project: https://github.com/freesoftwarefactory/parse-multipart#readme
+// Definitions by: pharrukh <https://github.com/pharrukh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export interface ParsedFile {
```

Comment:
I'd say this is not a file, but a part (ParsedPart), which here is always a file. 
The readme never mentionds file, just parts/subpart, which is more aligned with specs.
Up to you, you can resolve this one on your discrietion